github/commitstatus_test.go

```diff
@@ -55,17 +74,23 @@ func TestGitHubStatusSuccessMockAPI(t *testing.T) {
 		}
 		ts := httptest.NewServer(http.HandlerFunc(handler))
 		defer ts.Close()
-
 		target := github.Target{
```

Comment:
jitter is implicitly set to 0 here. Do we want to be explicit about it?

Reply:
See thread just above (https://github.com/Pix4D/cogito/pull/134#discussion_r1288255313)